libs/platform/user/nmr_um.cpp

```diff
@@ -46,6 +55,10 @@ NmrProviderDetachClientComplete(_In_ HANDLE nmr_binding_handle)
 NTSTATUS
 NmrWaitForProviderDeregisterComplete(_In_ HANDLE nmr_provider_handle)
 {
+    if (ebpf_fault_injection_inject_fault()) {
```

Comment:
I don't think it makes sense to fail NmrWaitForProviderDeregisterComplete with out of memory.